### ISSUE TITLE
Status bar contributions: allow `accessibilityInformation` and `tooltip`

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#183772]
Testing #183545

Looks like I cannot put in `accessibilityInformation` and `tooltip` statically. At least for accessibility I think we should provide it and maybe even should enforce it (?) to ensure people are building accessible extensions out of the box. Though our accessible defaults are probably already taken good care of it.
